File: app/src/main/java/dev/ragnarok/fenrir/model/EditingPostType.java
package dev.ragnarok.fenrir.model;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;


@IntDef({EditingPostType.DRAFT, EditingPostType.TEMP})
@Retention(RetentionPolicy.SOURCE)
public @interface EditingPostType {
    int DRAFT = 2;
    int TEMP = 3;
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/model/EditingPostType.kt
package dev.ragnarok.fenrir.model

import androidx.annotation.IntDef

@IntDef(EditingPostType.DRAFT, EditingPostType.TEMP)
@Retention(AnnotationRetention.SOURCE)
annotation class EditingPostType {
    companion object {
        const val DRAFT = 2
        const val TEMP = 3
    }
}